clusters/k8s-infra-dev-cluster-turnup/11-pool-1-configuration.tf

```diff
@@ -30,9 +30,15 @@ resource "google_container_node_pool" "pool-1" {
     max_node_count = 20
   }
 
+  // Set a realistic limit on pods per node, including daemonsets
+  max_pods_per_node = 32
```

Comment:
What's the reason for a non-GKE default?

Reply:
It economizes IP usage, but that's not a problem for us, so I will remove it.